pages/my-orders.js

```diff
@@ -9,12 +9,40 @@ export default function Orders() {
   const [orders, setOrders] = useState([])
   const headers = ['Order Date', 'Total', 'Payment Method']
 
+  const fetchPaymentTypes = async (url) => {
```

Comment:
so far convention in this project is to handle fetch requests in a separate data folder. I think its a good idea to follow that pattern for consistency and maintainability

Reply:
fetchPaymentTypes seems like its just a wrapper for getSinglePaymentType in this implementation, also, i suggest moving the data manipulation functions to the data folder.